src/visitor/import-export.js

```diff
@@ -123,14 +126,25 @@ class ImportExportVisitor extends Visitor {
     const { declaration } = node
     const { id, type } = declaration
 
-    if (id && (type === "FunctionDeclaration" ||
-               type === "ClassDeclaration")) {
+    if (type === "FunctionDeclaration" || (id && type === "ClassDeclaration")) {
+      const name = id ? id.name : encodeId("default")
```

Comment:
I didn't find a way of generatin an unique identifier. Is `default` + `ZWJ` ok?

Reply:
@nicolo-ribaudo
> I couldn't find a way of generating a unique identifier. Is `default` + `ZWJ` ok?

It's OK for this PR. I will follow up with a safe-name helper like [reify has](https://github.com/benjamn/reify/blob/3c002d12c32fa0be9bbbf648cb3b51777a6646d4/lib/import-export-visitor.js#L756-L761).

Reply:
Ok, so this is kinda neat. In making the safe-name helper I can just keep stacking ZWJs onto the name until there's no longer a conflict :yum: